README.md

```diff
@@ -0,0 +1,15 @@
+---
+layout: default
+---
+
+## Snapshots of documentation
+
+The current development work and git history of documentation is in the `dafny-lang/dafny` repository, in the `docs`
+folder.
+
+This `dafny-lang/dafny.github.io` repo is the target for `https://dafny.org` and is the entry point to all the user-facing
+documentation about Dafny.
+What is here are _snapshots_ of the Dafny documentation (the docs folder mentioned above) at the time of each release
+(beginning wwith 3.9.0). Each snapshot is in its own folder, e.g. dafny.org/vX.Y.Z, with `latest` 
+containing the most recent.
+
```

Comment:
Lets add a sentence about also having a separate copy of `index.html` and dependent files at the top level.

Reply:
Fixed.